force_wfmanager/ui/setup/tests/test_workflow_tree.py

```diff
@@ -137,6 +143,55 @@ def test_mco_kpis_selected(self):
         self.assertIsNone(self.system_state.remove_entity)
         self.assertIsNone(self.system_state.entity_creator)
 
+    def test_transfer_parameters(self):
+
+        # create old model
+        old_factory = DummyMCOFactory(plugin={'id': 'bcy356', 'name': 'old'})
+        old_factory.parameter_factory_classes = [
+            RangedVectorMCOParameterFactory,
+            RangedMCOParameterFactory
+        ]
+        old_model = BaseMCOModel(factory=old_factory)
+        old_model.parameters = [
+            RangedMCOParameter(
+                initial_value=1.0,
+                factory=RangedVectorMCOParameterFactory(old_factory)
+            ),
+            RangedVectorMCOParameter(
+                initial_value=[1.0, 1.0],
+                factory=RangedMCOParameterFactory(old_factory)
+            )
+        ]
```

Comment:
Could we add an additional class and parameter here (`CategoricalMCOParameterFactory`/`CategoricalMCOParameter`?) in `old_model.parameter_factory_classes` and `old_model.parameters`, which will not be supported in `new_factory`?

I expect the rest of the tests to run the same in this case - `new_factory.parameters` will only contain the `RangedMCOParameter` and `RangedVectorMCOParameter` instances.